test-suite/consensus-tests/tests/basic.rs

```diff
@@ -278,7 +278,10 @@ fn test_store_txs_positions() {
     let locations = schema.transactions_locations();
     for (expected_idx, hash) in hashes.iter().enumerate() {
         let location = locations.get(hash).unwrap();
-        assert_eq!(expected_idx as u64, location.position_in_block());
+        assert_eq!(
+            u32::try_from(expected_idx).unwrap(),
```

Comment:
You can use `(0..).zip(hashes.iter())` to get the `u32` values generated instead of `usize` produced by `enumerate`.

With that, there will be no need in `unwrap`.

Reply:
Does it worth that ? I don't think so. In the end it's just a test.

Reply:
IMHO it doesn't worsen the readability, and lets us get rid of bogus `try_from` (since we know that the values can't exceed the `u32` limit and `unwrap`ping it anyway).
Despite of this being a test part, I think we should treat all our code with the same love and care :)

However, if you insist, you can leave this as is, I don't mind.

Reply:
Other option (if you don't like the `(0..)` going first) is to zip them in the other order: `hashes.iter().zip(0..)`. Then you'll just have to swap `expected_idx` and `hash`.